tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class SdkOnlyImageTests
+    {
+        private static ImageDescriptor[] SdkTestData = new ImageDescriptor[]
+        {
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_1903 },
+            new ImageDescriptor { Version = "4.7.1", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_1903 },
+        };
+
+        private readonly ImageTestHelper imageTestHelper;
+
+        public SdkOnlyImageTests(ITestOutputHelper outputHelper)
+        {
+            imageTestHelper = new ImageTestHelper(outputHelper);
+        }
+
+        public static IEnumerable<object[]> GetVerifySdkImagesData()
+        {
+            return ImageTestHelper.ApplyImageDataFilters(SdkTestData);
+        }
+
+        /// <summary>
+        /// Verifies the SDK images contain the expected targeting packs.
+        /// </summary>
+        [SkippableTheory("4.6.2", "4.7")]
+        [Trait("Category", "sdk")]
+        [MemberData(nameof(GetVerifySdkImagesData))]
```

Comment:
The `GetVerifySdkImagesData` name in this context is confusing.  Given the refactoring to split the tests apart by image type, it feels like the `GetVerify*ImagesData` methods of each test class could be simplified to `GetImageData`.  Likewise the `*TestData` members could be renamed to `ImageData`.